### PR TITLE
feat: add generate conda forge package meta script

### PR DESCRIPTION
**Pull request by floralatin**

### CODE REVIEW

**Comment by floralatin** [comment]
And please review the [commit](https://github.com/jina-ai/staged-recipes/commit/a45302df8e5adeb6e0f20f2f8819c24023ac1d3a)

**Review by nomagick — APPROVED** [review]
LGTM if the yaml needs to be preserved in git.

**Comment by floralatin** [comment]
> LGTM if the yaml needs to be preserved in git.

Yaml is to be preserved in git. 
And  [conda-forge/staged-recipes](https://github.com/conda-forge/staged-recipes) will auto trace the version of `jina-hubble-sdk` after conda-forge merge this`[ pr](https://github.com/conda-forge/staged-recipes/pull/20927)`

File: conda/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{% set name = "jina-hubble-sdk" %}
+{% set version = "0.2.0" %}
```
Comment (mapleeit) [inline conda/meta.yaml:2]: Is this version correct?

File: conda/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jina-hubble-sdk-{{ version }}.tar.gz
+  sha256: bab9b7b4d8c5965e099e07effd92cc0607b56eead0b4d26d6e5b473cb93daf8f
```
Comment (mapleeit) [inline conda/meta.yaml:10]: Shall we convert this fixed sha256 to a template variable?

File: conda/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    - hubble
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/jina-ai/hubble-client-python/
+  summary: SDK for Hubble API at Jina AI.
+  dev_url: https://github.com/jina-ai/hubble-client-python/
```
Comment (mapleeit) [inline conda/meta.yaml:40]:
```suggestion
  home: https://github.com/jina-ai/jina-hubble-sdk/
  summary: SDK for Hubble API at Jina AI.
  dev_url: https://github.com/jina-ai/jina-hubble-sdk/
```

File: scripts/generate-conda-recipe.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+import sys
+
+root_path = 'conda'
+package_name = 'jina-hubble-sdk'
+tag = None
+
+def generate_meta_yaml_from_pypi(package_name: str, tag: str or None):
+    """Generate conda meta yaml from pypi"""
+
+    maintainers = 'mapleeit nomagick delgermurun Andrei997 floralatin'
```
Comment (mapleeit) [inline scripts/generate-conda-recipe.py:12]: Is it necessary to have this info twice both here and in `meta.yml`?

File: conda/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+  host:
+    - python >=3.7
+    - setuptools >=18.0
+    - wheel
+    - pip
+  run:
+    - python >=3.7
+    - requests ==2.27.1
+    - aiohttp ==3.8.1
+    - rich >=12.0.0
```
Comment (mapleeit) [inline conda/meta.yaml:27]: Is this mandantory because we already have `requirements.txt`, maintaining them in two places is hard

**Review by mapleeit — CHANGES_REQUESTED** [review]

File: conda/meta.yaml (hunk 2 above)
Comment (floralatin) [reply conda/meta.yaml:10]: > Shall we convert this fixed sha256 to a template variable?
No, it is  automatically created by running` python generate-conda-recipe.py`

File: conda/meta.yaml (hunk 3 above)
Comment (floralatin) [reply conda/meta.yaml:40]: Maybe we should convert `hubble-client-python` to `jina-hubble-sdk`
 the following places in the file `setup.py `

[_project_urls ](https://github.com/jina-ai/jina-hubble-sdk/blob/main/setup.py#:~:text=%5D-,_project_urls,-%3D%20%7B)
[_download_url](https://github.com/jina-ai/jina-hubble-sdk/blob/main/setup.py#:~:text=hubble%2Dclient%2Dpython/%27-,_download_url,-%3D%20%27https%3A//github)

[_url](https://github.com/jina-ai/jina-hubble-sdk/blob/main/setup.py#:~:text=)-,_url,-%3D%20%27https%3A//github)

File: conda/meta.yaml (hunk 5 above)
Comment (floralatin) [reply conda/meta.yaml:27]: This is not mandatory. We just recorded the information of the first version of meta.yaml, we don't need to keep track of it.
And [conda-forge/staged-recipes](https://github.com/conda-forge/staged-recipes) will auto trace the version of jina-hubble-sdk after conda-forge merge this https://github.com/conda-forge/staged-recipes/pull/20927

File: scripts/generate-conda-recipe.py (hunk 4 above)
Comment (floralatin) [reply scripts/generate-conda-recipe.py:12]: This info  `maintainers = 'mapleeit nomagick delgermurun Andrei997 floralatin` in meta.yml,
 It is our configuration and the result of running `python generate-conda-recipe`.

**Review by mapleeit — APPROVED** [review]

**Pull request merged**